src/main/java/com/frameworkium/core/ui/annotations/ForceVisible.java

```diff
@@ -8,5 +8,11 @@
 
     /** Default value. */
     String value() default "";
+
+    /**
+     * If checking for visibility of a list of elements, setting a value will only check for visibility of the first n
```

Comment:
I prefer a line length of ~80 chars, mainly because I can only fit ~100 comfortably on my laptop and even this github diff only displayed ~105 chars.

Reply:
you need a bigger laptop then ;)

I will shorten the lines - this would be a good use case for a change to the checkysyle config although better in a separate branch as it causes 200+ errors